mod-pubsub-server/src/test/java/org/folio/rest/impl/PublishersApiTest.java

```diff
@@ -332,4 +286,14 @@ private EventDescriptor postEventDescriptor(EventDescriptor eventDescriptor) {
     assertThat(postResponse.statusCode(), is(HttpStatus.SC_CREATED));
     return new JsonObject(postResponse.body().asString()).mapTo(EventDescriptor.class);
   }
+
+  private void postDeclarePublisher(PublisherDescriptor publisherDescriptor) {
+    RestAssured.given()
+      .spec(spec)
+      .body(JsonObject.mapFrom(publisherDescriptor).encode())
+      .when()
+      .post(EVENT_TYPES_PATH + DECLARE_PUBLISHER_PATH)
+      .then()
+      .statusCode(anyOf(is(HttpStatus.SC_CREATED), anyOf(is(HttpStatus.SC_INTERNAL_SERVER_ERROR))));
```

Comment:
Why did it changed to this from `.statusCode(HttpStatus.SC_CREATED);` ?

Reply:
error is recovered as we agreed